internal/accrual/internal/config/config.go

```diff
@@ -0,0 +1,86 @@
+package config
+
+import (
+	"flag"
+	"os"
+)
+
+type Config struct {
+	RunAddress      string
+	DatabaseAddress string
+	LoggerLevel     string
+	MigrationsPath  string
+	PublicKey       string
+}
+
+var cfg = &Config{}
+
+func GetConfig() *Config {
+	return cfg
+}
+
+func InitConfig() *Config {
+	runAddress := flag.String(
+		"a",
+		os.Getenv("RUN_ADDRESS"),
+		"Enter bind address for serve auth server as ip_address:port. Or use RUN_ADDRESS env")
+
+	databaseAddress := flag.String(
+		"d",
+		os.Getenv("DATABASE_URI"),
+		"Enter address exec http server as postgres://username:password@hostname:portNumber/databaseName?sslmode=disable. Or use DATABASE_URI env")
+
+	loggerLevel := flag.String(
+		"l",
+		os.Getenv("LOGGER_LEVEL"),
+		"Enter logger level as Warn. Or use LOGGER_LEVEL env")
+
+	migrationsPath := flag.String(
+		"m",
+		os.Getenv("MIGRATIONS_PATH_AUTH"),
+		"Enter migrations path. Or use MIGRATIONS_PATH_AUTH env")
+	publicKey := flag.String(
+		"publicKey",
+		`-----BEGIN PUBLIC KEY-----
+			MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA3vebIWQERuzgn0T/70ta
```

Comment:
It would be better if you load it from file or embedded filesystem

Reply:
Yes I agree. In addition, the best practice is to obtain a secret from the intelligence agency "Secret Management". But in an educational course, I think that's enough. However, we will not change paired keys, using them by default to demonstrate asynchronous cryptography.